content/hello-world-program.md

```diff
@@ -30,23 +30,29 @@ Rust organizes code using what is collectively referred to as the “module syst
 
 This includes:
 
-- **Packages** - A Cargo feature that lets you build, test, and share crates
-- **Crates** - A tree of modules that produces a library or executable
-- **Modules** and **use** - Let you control the organization, scope, and privacy of paths
-- **Paths** - A way of naming an item, such as a struct, function, or module
+- **Modules** - A module separates code into logical units to provide isolated namespaces for organization, scope, and privacy of paths
+- **Crates** - A crate is either a library or an executable program. The source code for a crate is usually subdivided into multiple modules.
+- **Packages** - A package contains a collection of crates as well as a manifest file for specifying metadata and dependencies between packages
 
-For this lesson, we’ll focus on using crates, modules, and paths.
+Throughout this lesson, we’ll focus on using crates and modules.
 
-### Pathways and scope
+### Paths and scope
 
-Crates in Rust contain modules that define functionality which can be shared with multiple projects. If we want to access an item within a module, then we need to know the path (like when we're navigating a filesystem).
+Crates in Rust contain modules that define functionality which can be shared with multiple projects. If we want to access an item within a module, then we need to know its "path" (like when we're navigating a filesystem).
 
-To use a module available within a crate, we'll need to:
+Think of the crate structure as a tree where the crate is the base and modules are branches, each of which can have submodules or items that are additional branches.
 
-1. Bring the module into the scope of our program
-2. Specify the item in the module tree using its path
+The path to a particular module or item is the name of each step from the crate to that module where each is separated by `::`. As an example, let's look at the following structure:
 
-Paths are brought into scope with the [use](https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html) keyword. In the example below, we bring into scope the `AccountInfo` struct from the `account_info` module within the `solana_program` crate. Bringing an item into scope this way allows us to use the item multiple times within our program without specifying the full path each time.
+1. The base crate is `solana_program`
+2. `solana_program` contains a module named `account_info`
+3. `account_info` contains a struct named `AccountInfo`
+   
```

Comment:
weird, my text editor shows two weird little orange dots here... not sure what they are but jsyk see pic:
![pr61-weirdOrangeDots](https://user-images.githubusercontent.com/18152993/174658455-1e2f810d-f2bf-4596-be68-65cec5221f97.png)